src/main/java/tz/go/moh/him/hdr/mediator/emr/Constants.java

```diff
@@ -0,0 +1,14 @@
+package tz.go.moh.him.hdr.mediator.emr;
+
+public class Constants {
```

Comment:
@cozej4 in the future, we might want to refactor error messages into a properties or resource file for simplicity